### PR TITLE
Rename kube_pv to odf_system_pvs

### PR DESCRIPTION
**Pull request by weirdwiz**
This commit renames the metric, in order to better indicate to the user
that the metric just represents the number of PVs created by ODF instead
of the total number of persistent volumes in a cluster.}|

https://issues.redhat.com/browse/RHSTOR-3472

### CODE REVIEW

**Review by simonpasquier — COMMENTED** [review]
same comment as for your other PR: you need to keep the old metric around.

File: manifests/0000_50_cluster-monitoring-operator_04-config.yaml [hunk 1]
```diff
@@ -327,11 +327,16 @@ data:
     #
     # owners: (@openshift/team-ocs-committers)
     #
-    # job:kube_pv:count is the total number of Persistent Volumes present in OCP cluster.
+    # job:kube_pv:count is the total number of Persistent Volumes created by ODF present in OCP cluster.
```
Comment (simonpasquier) [inline manifests/0000_50_cluster-monitoring-operator_04-config.yaml:330]: can you add a comment explaining that the metric is deprecated and replcaed by job:odf_system_pvs:count (also indicating from which version).

**Comment by weirdwiz** [comment]
PTAL @simonpasquier

**Comment by jan--f** [comment]
This lgtm, but will let @simonpasquier give final approval.

**Comment by jan--f** [comment]
@weirdwiz please run `make generate` and commit the changes.

**Comment by weirdwiz** [comment]
@simonpasquier can you tag this again?

**Pull request merged**